org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_manage-groups.scss

```diff
@@ -242,33 +242,44 @@
     }
 
     table {
-      tr {
-        td:nth-child(5) {
-          label[for*="group_status"] {
-            margin-top: 7px;
-          }
+      tr td:nth-child(5) label[for*="group_status"] {
+        margin-top: 7px;
+      }
+
+      tr td {
+        .font-italic {
+          font-style: normal;
         }
 
-        td {
-          .font-italic {
-            font-style: normal;
-          }
+        input[type="checkbox"]:checked + label[for]:after {
+          top: 8px;
         }
       }
     }
   }
 
-  .crm-block {
-     > .css_right {
-       .crm-hover-button {
-         color: $brand-primary !important;
-       }
-     }
-  }
+  .crm-block > .css_right .crm-hover-button {
+     color: $brand-primary !important;
+   }
 
   #DataTables_Table_0 {
     @include mx-expandable-table();
 
+    .crm-group-group_links {
+      text-align: right;
+
+      .btn-slide.crm-hover-button {
+        display: inline-block;
+        text-indent: -999999px;
```

Comment:
Do you need overflow:hidden in this? Some browsers just convert the text-indent to scroll so its important to have a wrapping element that hide any overflow.